Content.Client/UserInterface/GameHud.cs

```diff
@@ -121,6 +131,20 @@ public void AddTopNotification(TopNotification notification)
             _topNotificationContainer.AddChild(notification);
         }
 
+        public Texture GetHudTexture(string path)
+        {
+            var id = _configManager.GetCVar<int>("hud.theme");
+            var dir = (from theme in _prototypeManager.EnumeratePrototypes<HudThemePrototype>() where id.ToString() == theme.ID select theme.Path).FirstOrDefault();
+
+            if (string.IsNullOrEmpty(dir))
+            {
+                Logger.ErrorS("hud", "invalid HUD theme id {0}, using different theme",
+                    id);
+                dir = (from theme in _prototypeManager.EnumeratePrototypes<HudThemePrototype>() select theme.Path).FirstOrDefault();
+            }
+            return _resourceCache.GetTexture($"/Textures/Interface/Inventory/{dir}/{path}");
+        }
```

Comment:
This shouldn't really be here, have your HUD manager subscribe to `IConfigurationManager.OnValueChanged` and correct the value if it's ever changed to something incorrect. This also has the advantage of `invokeImmediately`, which lets you correct the value when the game starts, if it's incorrect!